addons/binding/org.openhab.binding.networkcamera/README.md

```diff
@@ -25,26 +25,54 @@ This binding currently supports following channels:
 |-----------------|--------------|----------------------------------------------------------------------------------------|
 | image           | Image        | Image received from network camera.                                                    |
 
+Binding also support custom Image channels, where matching filename can be configured.
+When image file is uploaded to FTP server, binding tries to find channel which filename match to upload image filename.
```

Comment:
When **an** image file.. server, **the** binding... find **the** channel ~~which~~ **whose** filename match**es** ~~to~~ **the** upload**ed**...